.changeset/loud-walls-compare.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/app-defaults': patch
```

Comment:
is this the correct plugin? looks like it's catalog that's changed

Reply:
sorry you're correct, this should be changed after i updated as per request